### PR TITLE
Added install of config file

### PR DESCRIPTION
**Pull request by scsides**

### CODE REVIEW

**Comment by jlaura** [comment]
Fixes #244

File: setup.py [hunk 1]
```diff
@@ -24,5 +24,6 @@
     packages=find_packages(),
     long_description="""\
     An Abstraction library for reading, writing and computing ephemeris data
-    """
+    """,
+    data_files=[('ale', ['ale/config.yml'])]
```
Comment (jessemapel) [inline setup.py]: Is there a reason why this uses data_files instead of package_data? I'm having a hard time with the setuptools docs.

Comment (Kelvinrr) [reply setup.py]: From looking around there doesn't seem to be much of a functional difference between the two other than package_data is used more frequently and is more recommended. No explanation as to why.

Comment (jessemapel) [reply setup.py]: Can this be moved to package_data then? It also looks less verbose.

**Comment by jessemapel** [comment]
This is good to merge once CI passes

File: setup.py (hunk 1 above)
Comment (scsides) [reply setup.py]: Done

**Pull request merged**